GLOSSARY.md

```diff
@@ -0,0 +1,29 @@
+1. kept sprite - A sprite that was in the DOM in the previous render, and is not scheduled for removal in this render. Should have initial and final DOM state. may have a counterpart.
```

Comment:
Glossary.md was more of me tidying up my thoughts so it's quite rough at the moment. I think some terms are defined in `ember-animated` and they work pretty similarly here, so could draw from there if/when we actually want to have a glossary.